src/MoBi.Presentation/Presenter/ChartPresenter.cs

```diff
@@ -75,6 +86,7 @@ public abstract class ChartPresenter : ChartPresenter<CurveChart, IChartView, IC
 
          _chartTemplatingTask = chartTemplatingTask;
          _dataRepositoryCache = new Cache<DataRepository, IMoBiSimulation>(onMissingKey: x => null);
+         _OutputMappingMatchingTask = OutputMappingMatchingTask;
```

Comment:
Casing is off here. Follow coding standard here

Reply:
done